26.md

```diff
@@ -0,0 +1,57 @@
+
+NIP-26
+======
+
+NostrAuth
+---------
+
+`draft` `optional` `author:saranshisatgit`
+
+The current approach for signed events is good but problematic on user side. Proposing a signed event with a `UIN` (User Identification Number). These UIN 
```

Comment:
I disagree that the current approach for signing events is problematic. On the contrary, it's very simple to use Schnorr signatures for creating identities.
There's many things left to interpretation because this NIP glosses over so many details.
This NIP pushes trust out of Nostr and to a third party, coupling the protocol to some blockchain which is wholly undesirable.
To me it sounds like you are trying to build a different protocol which isn't Nostr.

Reply:
The current approach is suitable but unsuitable for building use cases like reputation and relationships. This could be achieved if we use UIN.

This is not at all third party it uses an existing solution that is schnorr , it does not require any coupling with blockchain, user can store it on their website or any where as long its publically accessible